From c665a9c43f379f754b4ee25df2818b6c20c5346e Mon Sep 17 00:00:00 2001
From: Tao Bao <tbao@google.com>
Date: Mon, 20 May 2019 23:03:14 -0700
Subject: [PATCH] Build bootctrl.sdm710 with Android.bp.

Bug: 34254109
Test: TreeHugger
Test: Sideloading a package on bonito.
Change-Id: Ib29d901b44ad0ec079c3e979bfdcd467e1a18377
Merged-In: Ib29d901b44ad0ec079c3e979bfdcd467e1a18377
(cherry picked from commit aa97ef614cbfaca768fb382a659a764b13879a2b)
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index c5a9b16..6029054 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,5 +1,5 @@
 # Preset TARGET_USES_HARDWARE_QCOM_BOOTCTRL for existing platforms.
-ifneq ($(filter msm8996 msm8998 sdm710,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8996 msm8998,$(TARGET_BOARD_PLATFORM)),)
 TARGET_USES_HARDWARE_QCOM_BOOTCTRL := true
 endif
 
-- 
2.13.0

